fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/join/JoinLAsscomHelper.java

```diff
@@ -165,7 +166,8 @@ private Pair<List<NamedExpression>, List<NamedExpression>> getProjectExprs() {
     private LogicalJoin<GroupPlan, GroupPlan> newBottomJoin() {
         return new LogicalJoin(
                 bottomJoin.getJoinType(),
-                Optional.of(ExpressionUtils.and(newBottomJoinOnCondition)),
+                newBottomJoinOnCondition,
+                Optional.empty(),
```

Comment:
why all other conjuncts are empty? do we will lose some expression on other conjuncts such as t1.a > t2.b?

Reply:
add corresponding nonHashJoinConjuncts for bottom and top join.